lldb/tools/lldb-dap/IOStream.h

```diff
@@ -52,11 +51,11 @@ struct StreamDescriptor {
 struct InputStream {
   StreamDescriptor descriptor;
 
-  bool read_full(std::ofstream *log, size_t length, std::string &text);
+  bool read_full(llvm::raw_ostream *log, size_t length, std::string &text);
 
-  bool read_line(std::ofstream *log, std::string &line);
+  bool read_line(llvm::raw_ostream *log, std::string &line);
 
-  bool read_expected(std::ofstream *log, llvm::StringRef expected);
+  bool read_expected(llvm::raw_ostream *log, llvm::StringRef expected);
```

Comment:
I'm not sure why is this being changed, but could that be a separate patch as well?

Reply:
Reverted this change.

Reply:
(Note I'm not saying this isn't a good change. I'd be happy to review something like this as a separate patch. I just don't want to mix this in with the rest of the changes, as there's a lot of things going on here already.)